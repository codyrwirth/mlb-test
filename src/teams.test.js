/**
 * @jest-environment jsdom
 */

 import renderer from 'react-test-renderer';
 import Teams from './components/teams';
 import React, { Component } from 'react';
 import { shallow, render, mount } from 'enzyme';
 import { configure } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 
 // write unit tests for the teams Component
 describe('Teams Component', () => {
	   it('renders correctly', () => {
		 const tree = renderer.create(<Teams></Teams>).toJSON();
		 expect(tree).toMatchSnapshot();
   });
 }
 );
 
 