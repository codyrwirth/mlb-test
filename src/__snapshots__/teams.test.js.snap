// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teams Component renders correctly 1`] = `
<div
  className="toolbar"
>
  <div
    className="flex-container"
  />
  <div
    className="displayedData"
  />
</div>
`;
